src/coreclr/debug/dbgutil/machoreader.h

```diff
@@ -43,7 +44,8 @@ class MachOModule
 
     bool ReadLoadCommands();
     bool ReadSymbolTable();
-    void* GetAddressFromFileOffset(uint32_t offset);
+    uint64_t GetAddressFromFileOffset(uint32_t offset);
+    const char* GetSymbolName(int index);
```

Comment:
CI error: `error: address of stack memory associated with local variable 'result' returned` is due to `result.c_str()`. Since `SetName()` in this class (line 43) is also using std::string, we can use it here to make clang happy (and NRVO will likely kick in to avoid copies in release build):
```suggestion
    const std::string GetSymbolName(int index);
```

Reply:
Done.